tests/test_rio_mask.py

```diff
@@ -160,7 +164,9 @@ def test_mask_invalid_geojson(runner, tmpdir, pixelated_image_file):
     assert 'Invalid GeoJSON' in result.output
 
 
-
+@pytest.mark.xfail(
+    gdal_version.at_least('2.3'),
+    reason="Test only applicable to GDAL < 2.3")
```

Comment:
There's another breaking change in 2.3 with respect to ~identity transforms, or perhaps a fix of a bug we were dependent on.